integr8ly/integreatly-operator
# INTLY-5199: Add datasync templates installation

# What
The following changes creates the [datasync-server-app](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-http.yml) and [datasync-showcase-server](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-showcase.yml) as part of the RHMI installation. 

These templates are created with an ownerReference that points to the RHMI CR so that when this CR gets deleted both of the templates are cleaned up during uninstallation.

**Datasync Templates Version**: 0.9.4
**JIRA**: https://issues.redhat.com/browse/INTLY-5199

## Verification
1. Install RHMI using this branch
   - **Verify**: Ensure both of the templates mentioned above are created successfully.
   - **Verify**: Datasync product info and status should be displayed in the RHMI CR
2. Uninstall RHMI by removing the RHMI CR
   - **Verify**: Ensure both of the templates mentioned above are deleted
   - **Verify**: RHMI uninstallation completed successfully
